toolchain/install/install_filegroups.bzl

```
@@ -0,0 +1,132 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""Rules for constructing install information."""
+
+load("@rules_pkg//pkg:mappings.bzl", "pkg_attributes", "pkg_filegroup", "pkg_files", "pkg_mklink", "strip_prefix")
+load("symlink_helpers.bzl", "symlink_file", "symlink_filegroup")
+
+def install_filegroup(name, filegroup_target):
+    """Adds a filegroup for install.
+
+    Used in the `install_dirs` dict.
+
+    Args:
+      name: The base directory for the filegroup.
+      filegroup_target: A relative path for the symlink.
```

Not clear what "symlink" this is a target for?

Or was this supposed to read "A relative path for the filegroup"? If so, not sure I understand what that means.

---

Having finish reading, I think this is just a copy/paste. I'm pretty sure I understand how this is used, maybe:

```suggestion
      filegroup_target: The bazel filegroup target to install.
```